src/components/SharedComponents/UploadStatus/UploadStatus.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import { View } from "components/styledComponents";
+import * as React from "react";
+import CircularProgressBase from "react-native-circular-progress-indicator";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  color: string,
+  completeColor: string,
+  progress: number
+}
+
+const UploadStatus = ( { color, completeColor, progress }: Props ): React.Node => (
+  <View className="relative justify-center items-center">
```

Comment:
Can you add a default width and height 49 x 67 here. In the design specs every one of the different status icons has a box of these dimensions around it. I guess so that it fills a similar space than ObsStatus in a list item.